### ISSUE TITLE
Please publish Docker image

### ISSUE THREAD

**Issue by mingfang** [SoftInstigate/http-shell#4]
it would be great to run `http-shell` web interface inside Kubernetes .

**Comment by mkjsix** [comment]
Hi, 
Could you please explain the use case for running into Kubernetes? As this is a CLI, I don't get it.

**Comment by mingfang** [comment]
This section in the readme https://github.com/SoftInstigate/http-shell/blob/master/README.md?plain=1#L52
gave me the impression that this was web based.
I wasn't able to build locally to confirm.

**Comment by mkjsix** [comment]
@mingfang you are right, it's just that we have not used the tool ourselves that way so far. I'll give a look in the following days, it shouldn't take much effort but we are under pressure these days.
